Mlem/App/Views/Root/Tabs/Settings/InteractionBarEditor/InteractionBarEditorView+Views.swift

```diff
@@ -0,0 +1,329 @@
+//
+//  InteractionBarEditorView+Views.swift
+//  Mlem
+//
+//  Created by Eric Andrews on 2025-01-27.
+//
+
+import SwiftUI
+import Flow
+
+extension InteractionBarEditorView {
+    
+    // MARK: - Post Preview
+    
+    @ViewBuilder
+    var postPreview: some View {
+        VStack(alignment: .leading, spacing: Constants.main.standardSpacing) {
+            Group {
+                HStack(alignment: .top, spacing: 8) {
+                    RoundedRectangle(cornerRadius: Constants.main.smallItemCornerRadius)
+                        .fill(palette.accent.opacity(0.6))
+                        .frame(width: Constants.main.thumbnailSize, height: Constants.main.thumbnailSize)
+                        .overlay {
+                            Image(systemName: "mountain.2.fill")
+                                .font(.system(size: 23))
+                                .foregroundStyle(.white)
+                        }
+                    
+                    VStack(alignment: .leading, spacing: 5) {
+                        MockTextView()
+                            .frame(maxWidth: .infinity)
+                            .frame(height: 15)
+                        MockTextView()
+                            .frame(maxWidth: 200)
+                            .frame(height: 15)
+                    }
+                }
+            }
+            .opacity(0.75)
+            
+            interactionBar
+                .frame(height: Constants.main.barIconSize)
+                .padding(.horizontal, 2)
+                .padding(.vertical, Constants.main.barIconPadding)
+        }
+        .padding(Constants.main.standardSpacing)
+        .background(palette.background, in: .rect(cornerRadius: Constants.main.mediumItemCornerRadius))
```

Comment:
Doesn't work in dark mode

```suggestion
        .background(palette.secondaryGroupedBackground, in: .rect(cornerRadius: Constants.main.mediumItemCornerRadius))
```